cypress/integration/login.js

```diff
@@ -21,4 +21,12 @@ describe('Login functionality', function() {
         cy.get('.user-info__title').contains('TestCon User 9')
         cy.get('.main-nav').find('li').should('have.length', 2)
     })
+
+    it('Validate what date is selected as today', function() {
+
+        const date = new Date();
+        const currentDate = date.getDate()
+        cy.get('.tile__title').contains(currentDate)
```

Comment:
you should assert the selected day in calendar, not the side of the screen.

Reply:
Done